config/locales/en.yml

```diff
@@ -715,6 +723,7 @@ en:
         request_type:
           nomis_records: NOMIS records
           nomis_contact_logs: NOMIS contact logs
+          nomis_contact_lists: NOMIS contact lists
```

Comment:
Is this new value required. If so, maybe it can be added in a separate PR as it will require a migration. Probably best to remove for now from here and `BRANSTON_DATA_REQUEST_TYPES`